css/css-transitions/idlharness.html

```diff
@@ -8,16 +8,18 @@
 <script>
   "use strict";
 
-  promise_test(async () => {
-    const idl_array = new IdlArray();
-    const idl = await fetch("/interfaces/css-transitions.idl").then(r => r.text());
-    const cssom = await fetch("/interfaces/cssom.idl").then(r => r.text());
-    const html = await fetch("/interfaces/html.idl").then(r => r.text());
-    const dom = await fetch("/interfaces/dom.idl").then(r => r.text());
-    idl_array.add_idls(idl);
-    idl_array.add_dependency_idls(cssom);
-    idl_array.add_dependency_idls(html);
-    idl_array.add_dependency_idls(dom);
-    idl_array.test();
-  }, "Test IDL implementation of css-transitions API");
+  idl_test(
+    ['css-transitions'],
+    ['cssom', 'html', 'dom'],
+    idl_array => {
+      idl_array.add_objects({
+        TransitionEvent: ['new TransitionEvent("transition")'],
```

Comment:
It might be better to use an actual event name here instead of `transition`, e.g. `transitionend` (the most widely supported transition event)

Reply:
Something like "x" or "type" is also good in that it's clearer that the string doesn't matter. (I've seen some case where multiple instances that differ only by the event type have been tested, which won't do any good.)